package.json

```diff
@@ -158,9 +158,10 @@
     "watch": "tsc -watch -p ./",
     "postinstall": "node ./scripts/postinstall.js",
     "test": "node ./out/src/test/runTest.js",
+		"test:coverage": "c8 npm run test",
```

Comment:
spaces are used instead of tabs for other lines for indentation (even if using tabs is more accessible, keeping it coherent is more important I think)